UPGRADE.md

```diff
@@ -1,5 +1,16 @@
 # Upgrade
 
+## dev-develop
+
+### Snippets
+
+Snippet state has been removed and set default to published. Therefor all snippets has to be set to published by this
+running this command:
+
+```bash
+app/console sulu:upgrade:rc3:snippet-state
```

Comment:
Interferes with https://github.com/sulu-io/sulu/issues/819 ... But I guess we will also just remove this command before 1.0